src/hotspot/share/opto/library_call.cpp

```diff
@@ -7222,11 +7222,12 @@ bool LibraryCallKit::inline_counterMode_AESCrypt(vmIntrinsics::ID id) {
 
 //------------------------------get_key_start_from_aescrypt_object-----------------------
 Node * LibraryCallKit::get_key_start_from_aescrypt_object(Node *aescrypt_object) {
-#if defined(PPC64) || defined(S390)
+#if defined(PPC64) || defined(S390) || defined(RISCV)
```

Comment:
Hi, I think RISCV64 is more specific here for now.

Reply:
Done!